index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="welcome">
+            Welcome
```

Comment:
### Your Header
![navbar-fixable (copy)](https://user-images.githubusercontent.com/93039359/186411334-a87a34cb-4c37-40d1-9322-81cb7723d23c.png)

### Figma Header
![navbar-fixed](https://user-images.githubusercontent.com/93039359/186411566-25927951-5a9d-4da6-9d4c-1828ded0f4de.png)

- [ ] Please fix header to follow the Figma template :+1: I have given the quick fix below :arrow_down:
### Quick Fix
#### 1.
![navbar](https://user-images.githubusercontent.com/93039359/186410973-3e2870c5-1256-4b35-8de3-8bf87da07366.png)